RELEASE.md

```diff
@@ -1,6 +1,7 @@
 # Upcoming Release 0.19.4
 
 ## Major features and improvements
+* Cookiecutter errors are shown in short format without the `--verbose` flag.
```

Comment:
Are the error now shown partly for all CLI commands or just `kedro new`?

Reply:
Now they're shown in short format for all commands but just for `cookiecutter errors`. So instead of `Error: Failed to generate project when running cookiecutter.` we will also get one line with a real error.